var/spack/repos/builtin/packages/malt/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Malt(CMakePackage):
+    """
+    MALT is a memory profile tool to track mallocs in a C/C++ application and report
+    allocation information (lifetime, sizes...) in a friendly web graphical interface
+    by annotating the source code and proving charts.
+    """
+
+    # Project infos
+    homepage = "https://memtt.github.io/malt"
+    url = "https://github.com/memtt/malt/archive/v1.2.1.tar.gz"
+    maintainers = ["svalat"]
```

Comment:
Missed this.  The syntax for this changed recently.  A quick change and I'll merge it.

```suggestion
    maintainers("svalat")
```